[New Game] : Ludo game #2099

PriyankaTamhankar01 opened the pull request:
## PR Description 📜

Please include summary related to the issue you have fixed and describe your PR in brief over here by specifying the issue number on which you were working below
## [New Game] : Ludo game #2099

<hr>
 
## Mark the task you have completed ✅

<!----Please delete options that are not relevant. In order to tick the check box just but x inside them for example [x] like this----->

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.


<hr>

## Add your screenshots(Optional) 📸
![Ludo King Game](https://github.com/kunjgit/GameZone/assets/90240653/6dc5e629-acc4-4541-857b-8c96312a10be)



--- 
<br>

## Thank you soo much for contributing to our repository 💗

kunjgit commented:
Thank you @PriyankaTamhankar01 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

kunjgit reviewed (changes requested):
Hello @PriyankaTamhankar01,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

kunjgit reviewed (changes requested):
Hello @PriyankaTamhankar01,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

kunjgit reviewed (changes requested):
Hello @PriyankaTamhankar01,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

PriyankaTamhankar01 commented:
[Shadow_PokeGuess](https://github.com/kunjgit/GameZone/tree/main/Games/Shadow_PokeGuess) Is repeated in 286 & 288

kunjgit reviewed (changes requested):
Hello @PriyankaTamhankar01,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

PriyankaTamhankar01 commented:
Hello! I have completed the requested tasks :

>-  I have modified the main README.md file as per your request.
>-  I have added an image to the assets folder.

Please let me know if there's anything else I can assist you with. Thank you!"

![image](https://github.com/kunjgit/GameZone/assets/90240653/60e418e8-6fbc-4f32-b366-f69e31f92566)

![image](https://github.com/kunjgit/GameZone/assets/90240653/dc85247f-4c7b-43dc-b56b-2103a40701d3)

kunjgit reviewed (changes requested):
Hello @PriyankaTamhankar01,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

kunjgit reviewed (changes requested):
Hello @PriyankaTamhankar01,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀
